service_configuration_lib/spark_config.py

```diff
@@ -271,6 +271,12 @@ def _get_dra_configs(spark_opts: Dict[str, str]) -> Dict[str, str]:
         spark_opts, 'spark.dynamicAllocation.cachedExecutorIdleTimeout',
         str(DEFAULT_DRA_CACHED_EXECUTOR_IDLE_TIMEOUT),
     )
+    log.warning(
+        f'\nSetting spark.dynamicAllocation.cachedExecutorIdleTimeout as {DEFAULT_DRA_CACHED_EXECUTOR_IDLE_TIMEOUT}. '
```

Comment:
Nit: Maybe also add a reason of increasing. Something like, "If your job is performing bad as the cached data was lost, try increasing the cachedTimeoutValue.

Separate: It would also be nice to know the error one gets or an indication in logs, on how one can that missed cached blocks were expected to be used later. An experiment with low value of timeout and observing could a starting point.